docs/assets/js/examples.js

```diff
@@ -101,6 +101,7 @@ $(document).ready(function () {
       const $activeCodeSnippet = $(activeCodeSnippet)
       const exampleTarget = $(this).data('example')
       const fileId = $(this).data('target')
+      const rangeId = $(this).data('range-id')
```

Comment:
Instead of `range-id`, should this be called `snippet-id`, here and throughout the PR, given that you use `snippet-tag-xxx` everywhere?

Reply:
Renamed all include params and local var to match "snippet" concept.